--- kalibr/aslam_offline_calibration/kalibr/python/kalibr_calibrate_cameras
+++ kalibr/aslam_offline_calibration/kalibr/python/kalibr_calibrate_cameras
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 print("importing libraries")
 import rosbag
+import cv2
 import cv_bridge
 
 import sm
@@ -11,7 +12,6 @@ import aslam_cv_backend as acvb
 import kalibr_common as kc
 import kalibr_camera_calibration as kcc
 
-import cv2
 import os
 import numpy as np
 import multiprocessing
--- kalibr/aslam_offline_calibration/kalibr/python/kalibr_calibrate_rs_cameras
+++ kalibr/aslam_offline_calibration/kalibr/python/kalibr_calibrate_rs_cameras
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 print("importing libraries")
 import rosbag
+import cv2
 import cv_bridge
 
 import sm
@@ -12,7 +13,7 @@ import kalibr_common as kc
 import kalibr_camera_calibration as kcc
 import kalibr_rs_camera_calibration as rscc
 
-import cv2
+
 import os
 import numpy as np
 import pylab as pl
--- kalibr/aslam_offline_calibration/kalibr/python/kalibr_camera_validator
+++ kalibr/aslam_offline_calibration/kalibr/python/kalibr_camera_validator
@@ -2,10 +2,10 @@
 import rospy
 from std_msgs.msg import String
 from sensor_msgs.msg import Image, CompressedImage
+import cv2
 from cv_bridge import CvBridge, CvBridgeError
 import message_filters
 
-import cv2
 import sm
 import aslam_cv as acv
 import aslam_cv_backend as acvb
--- kalibr/aslam_offline_calibration/kalibr/python/kalibr_common/ImageDatasetReader.py
+++ kalibr/aslam_offline_calibration/kalibr/python/kalibr_common/ImageDatasetReader.py
@@ -1,5 +1,5 @@
-import cv_bridge
 import cv2
+import cv_bridge
 import rosbag
 import os
 import numpy as np
